lib/debug/session.rb

```diff
@@ -185,7 +186,11 @@ def wait_command_loop tc
 
     def wait_command
       if @initial_commands.empty?
-        line = @ui.readline
+        line =  if ENV['RUBY_DEBUG_TEST_MODE']
+                  @ui.readline(@internal_info)
```

Comment:
Changing parameters of `@ui.readline` is nod good idea because `#readline` is defined for each UI classes.
Why don't you write a `@ui.puts info` first?

Reply:
That's a good idea.

Reply:
https://github.com/ruby/debug/pull/48/files#diff-c87adef8ace662a03249d206e6812162256bbcecabe18cd51861d1e1cd9cdfcaR189